base/applications/mspaint/winproc.cpp

```diff
@@ -235,6 +261,19 @@ LRESULT CMainWindow::OnCreate(UINT nMsg, WPARAM wParam, LPARAM lParam, BOOL& bHa
 LRESULT CMainWindow::OnDestroy(UINT nMsg, WPARAM wParam, LPARAM lParam, BOOL& bHandled)
 {
     GetWindowPlacement(&(registrySettings.WindowPlacement));
+
+    if (s_pHtmlHelpW)
```

Comment:
You could still call DoHtmlHelp here (see also the review comment I'm adding in that function above).

Reply:
Only when once loaded.

Reply:
> Only when once loaded.

But that detail should be handled by the implementation of DoHtmlHelp.